optaplanner-examples/src/main/java/org/optaplanner/examples/batchscheduling/score/BatchSchedulingIncrementalScoreCalculator.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.optaplanner.examples.batchscheduling.solver.score;
+package org.optaplanner.examples.batchscheduling.score;
```

Comment:
We don't have the `ConstraintProvider` for this example yet?

Reply:
No. This one was provided by the community and we only have easy/incremental here. Those are somewhat difficult to reverse-engineer well enough to convert to a constraint stream, and therefore I did not bother.